createdisk.sh

```diff
@@ -247,15 +247,17 @@ if [ -z $random_string ]; then
 fi
 VM_PREFIX=${CRC_VM_NAME}-${random_string}
 
-# First check if cert rotation happened.
-# Initial certificate is only valid for 24 hours, after rotation, it's valid for 30 days.
-# We check if it's valid for more than 25 days rather than 30 days to give us some
-# leeway regarding when we run the check with respect to rotation time
-if ! ${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} -- sudo openssl x509 -checkend 2160000 -noout -in /var/lib/kubelet/pki/kubelet-client-current.pem; then
-    echo "Certs are not yet rotated to have 30 days validity"
-    exit 1;
+# Only validate the cert expire time if VALIDATE_CERT is set to true
+if [ "${VALIDATE_CERT:-false}" = true ]; then
+    # First check if cert rotation happened.
+    # Initial certificate is only valid for 24 hours, after rotation, it's valid for 30 days.
+    # We check if it's valid for more than 25 days rather than 30 days to give us some
+    # leeway regarding when we run the check with respect to rotation time
+    if ! ${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} -- sudo openssl x509 -checkend 2160000 -noout -in /var/lib/kubelet/pki/kubelet-client-current.pem; then
+        echo "Certs are not yet rotated to have 30 days validity"
+        exit 1;
```

Comment:
I'd keep the check so that we have a warning in the logs that the certificates did not rotate. And I'd actually consider reusing `SNC_NON_FATAL_PREFLIGHT_CHECKS` which is introduced in the preflight commit as the cert validity check is arguably some kind of preflight check.
        fi`